<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="statuslabel">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Desconectado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="horizontalGroupBox">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Procure sua m√∫sica: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leMusicSearchName"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbSearch">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="twMusicSearchResults">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="horizontalGroupBox">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Crie sua playlist:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPlaylistName"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbAdd">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewPlaylist">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="horizontalGroupBox">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbAddMusic">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlay">
         <property name="whatsThis">
          <string>teste</string>
         </property>
         <property name="accessibleName">
          <string>teste</string>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPause">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlaylistSave">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlaylisttLoad">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemove">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemoveMusic">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWeb_API">
    <property name="title">
     <string>Spotify</string>
    </property>
    <addaction name="actionGrant"/>
    <addaction name="actionGet_Playlists"/>
    <addaction name="actionGet_User_Information"/>
   </widget>
   <addaction name="menuWeb_API"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGrant">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
  <action name="actionGet_Playlists">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get Playlists</string>
   </property>
   <property name="toolTip">
    <string>Query Playlists from Webapi</string>
   </property>
  </action>
  <action name="actionGet_User_Information">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get User Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
